build.sbt

```diff
@@ -882,6 +899,20 @@ lazy val tests =
             Seq((Test / sourceDirectory).value / "scala-2.12+")
           case _ => Nil
         }
+      },
+      // temp hack
+      Test / unmanagedSourceDirectories ++= {
```

Comment:
The same settings should be applied to the JVM project to make sure that our tests our compliant with the JVM runtime. It would be the easiest to move this logic to the `sharedTestSource` method. In fact, we can also move `require-jdk*` directories inside the `shared` directory.

Reply:
Yes, I created a setting and added it to both native and jvm tests.
Now the following works:
```scala
sbt:scala-native> testsJVM/testOnly *CharacterTest*
[info] Test run started
[info] Test org.scalanative.testsuite.javalib.lang.CharacterTestOnJDK11.toStringCodePoint started
[info] Test run finished: 0 failed, 0 ignored, 1 total, 0.039s
```